packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -620,6 +621,17 @@ const ruleset: any = {
         field: "type",
       },
     },
+    NoErrorCodeResponses: {
+      description: "Responses must not have error codes. All errors must be surfaced using `default`.",
```

Comment:
>Responses must not have error codes [](http://example.com/codeflow?start=20&length=35)

Invalid status code specified. Please refer the documentation for the allowed set.